services/aur/aur.service.js

```diff
@@ -168,8 +174,85 @@ class AurVersion extends BaseAurService {
   }
 }
 
+class AurMaintainer extends BaseAurService {
+  static get category() {
+    return 'social'
+  }
+
+  static get route() {
+    return {
+      base: 'aur/maintainer',
+      pattern: ':packageName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'AUR maintainer',
+        namedParams: { packageName: 'google-chrome' },
+        staticPreview: this.render({ maintainer: 'luzifer' }),
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'maintainer' }
+  }
+
+  static render({ maintainer }) {
+    return { message: maintainer, color: 'blue' }
+  }
+
+  async handle({ packageName }) {
+    const json = await this.fetch({ packageName })
+    return this.constructor.render({ maintainer: json.results.Maintainer })
+  }
+}
+
+class AurLastModified extends BaseAurService {
+  static get category() {
+    return 'activity'
+  }
+
+  static get route() {
+    return {
+      base: 'aur/last-modified',
+      pattern: ':packageName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'AUR last modified',
+        namedParams: { packageName: 'google-chrome' },
+        staticPreview: this.render({ date: new Date().getTime() }),
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'last modified' }
+  }
+
+  static render({ date }) {
+    const color = ageColor(date)
+    const message = formatDate(date)
+    return { color, message }
+  }
+
+  async handle({ packageName }) {
+    const json = await this.fetch({ packageName })
+    const date = 1000 * parseInt(json.results.LastModified)
```

Comment:
I don't suppose there's a spec/doc for this API anywhere is there? If so might be helpful to include that link in an inline comment for future eyes

Reply:
There's am arch [wiki page](https://wiki.archlinux.org/index.php/Aurweb_RPC_interface) it has a reference of the expected parameters. But no details regarding data-types.